setup.sh

```diff
@@ -317,7 +317,7 @@ install_hooks() {
 
 install_our_lovely_cli() {
   cd "$DEVTOOLS_DIR/our-lovely-cli"
-  npm install
+  make
```

Comment:
On second thought, we should have an explicit make target here, rather than just depending on the first one being the one to install.  I'm not even sure what it should be, looking at the (confusing) Makefile.   `make yarn`?  `make all`?  You may want to ask in `#github-prs`.

Reply:
@csilvers we're having problems with our `Makefile`, and it's been on my list to reach out to you to chat w you (when you have time), as everyone says you're the "make expert" here. I wonder if we should solve that first, since the solution _could_ be to just get rid of the `Makefile` altogether...

Reply:
@lillialexis am happy to chat with you!  My calendar is accurate if you want to find time tomorrow.

Reply:
@marekweb please use `make all`

Reply:
oh, wait, maybe `make all` doesn't do `yarn build`, but I'll fix that in OLC momentarily...

Reply:
Alright, I updated it to `make all` assuming that will be the final command.